pkg/cmd/repo/autolink/list.go

```diff
@@ -0,0 +1,123 @@
+package autolink
```

Comment:
To follow our existing patterns, this should probably go under a `list` package in a subdirectory. That said, I can see the value in not adding that abstraction quite yet, as there's only a single subcommand to autolink being introduced in this PR, and instead adding it in the next PR that introduces the second subcommand.

What do you think?

Reply:
I went back and forth on this myself. I noticed `repo/deploy-key` uses a multi-package structure. Elsewhere (outside of `repo/`), the `label` command uses a flat, single-package structure. Given that ambiguity and the fact that—as you note—there is only one subcommand in this dir for now, I came down on the side of "flat".

However, prompted by your question, I am now noticing that within `repo/`, there are actually _three_ commands with subcommands—`deploy-key`, `gitignore`, and `license`—and they all use subdirectories. For this reason I think it makes sense to introduce the abstraction now. I plan to develop the other subcommands soon (if no one beats me to it), and if for some reason that doesn't happen, I don't want to muddy the waters as to the preferred structure.

Reply:
I implemented this change via 955bd6b2480ac045105a55cd5480fc91b247f93c.

If the `get` wants to reuse the `autolink` struct type, we'll have to move that type out of the `list` package at that point.